src/img/index.js

```diff
@@ -0,0 +1,6 @@
+export {default as g_bw} from "./g_bw.jpeg";
+export {default as j_color} from "./j_color.jpeg";
+export {default as jg_bw} from "./J&G_0147.jpeg";
```

Comment:
I'd avoid special characters (`&`) in file names. Might cause issues in unexpected ways.